src/Util/YamlSourceManipulator.php

```diff
@@ -714,7 +714,7 @@ private function updateContents(string $newContents, array $newData, int $newPos
     private function convertToYaml($data)
     {
         $indent = $this->depth > 0 && isset($this->indentationForDepths[$this->depth])
-            ? $this->indentationForDepths[$this->depth] / $this->depth
```

Comment:
💡 This division might evaluate to a float value. The problem is that `$indent` is implicitly cast to an integer later and PHP 8.1 will complain if that implicit cast would result in a loss of precision. Since the loss in precision is actually desired here, I'm performing an integer division instead which will always yield an integer.